src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -12,7 +13,8 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic;
 
-internal class RazorSemanticTokensLegend
+[Export(typeof(RazorSemanticTokensLegendService))]
+internal sealed class RazorSemanticTokensLegendService
```

Comment:
I decided an interface here was just going to be more annoying than its worth, because a) there are a bunch of properties that are just silly to maintain in two places, and b) there are static strings that would have to live on the concrete service anyway, so anything really using the interface would also have to use the concrete impl.